docs/api.md

```diff
@@ -59,6 +59,7 @@
 * `.request` - **Request**
 * `.cookies` - **Cookies**
 * `.history` - **List[Response]**
+* `.elapsed` - **timedelta**
```

Comment:
Building on @sethmlarson's comment…

> Btw when we document this let's put a warning to use timedelta.total_seconds() instead of .seconds. This is a common issue that new users of the datetime module hit.

I think it's a fair point. Granted, the developer interface documentation is currently rather scarce — only the attribute name and its type are listed. But we can probably start breaking the pattern and explain more what each attribute corresponds to.

What would you think about adding this usage hint here, along with a description of what `.elapsed` measures exactly?